reporter/html.js

```diff
@@ -365,6 +366,15 @@ export function escapeText( s ) {
 			dropDownList = document.createElement( "ul" ),
 			dirty = false;
 
+		const options = {
+			includeScore: true,
+
+			// Search in `author` and in `tags` array
```

Comment:
This comment doesn't seem to apply to our data set. Is it needed?

Reply:
Nope, removed :-).